game/scripts/npc/items/item_helm_of_the_dominator.txt

```diff
@@ -62,12 +62,12 @@
       "01"
 			{
 				"var_type"				"FIELD_INTEGER"
-				"bonus_stats"			"2 25 40 55 70"
+				"bonus_stats"			"2 20 30 45 60"
 			}
 			"02"
 			{
 				"var_type"				"FIELD_INTEGER"
-        "attack_speed_aura"   "20 35 60 80 100"
+        "attack_speed_aura"   "20 35 50 65 80"
```

Comment:
These changes plus your AC buffs are just going to make AC a better item. AC will give exaclty the same attackspeed as HoD for the user and will give a stronger aura for your team. 

In my opinion this item doesn't need nerfing as you know my stance on the lack off attack speed items in OAA. However if you really feel this item needs a nerf, buff the garbage regen aura and make the dominated creep stronger.